x-pack/plugins/infra/public/components/logging/log_analysis_setup/create_job_button.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButton, PropsOf } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React from 'react';
+import { MissingSetupPrivilegesToolTip } from './missing_setup_privileges_tooltip';
+
+export const CreateJobButton: React.FunctionComponent<
+  {
+    hasSetupCapabilities?: boolean;
+  } & PropsOf<typeof EuiButton>
+> = ({ hasSetupCapabilities = true, children, ...buttonProps }) => {
+  const button = (
+    <EuiButton isDisabled={!hasSetupCapabilities} {...buttonProps}>
+      {children ?? (
+        <FormattedMessage
+          id="xpack.infra.logs.analysis.createJobButtonLabel"
+          defaultMessage="Create ML jobs"
```

Comment:
For the default text here should it be job instead of jobs?

Reply:
I would say, it depends on whether we want to abstract away the fact that this module happens to only contain one job. :shrug: